src/components/TimeSeries/TimeSeries.scss

```diff
@@ -0,0 +1,7 @@
+.column-left {
```

Comment:
Use Tailwind instead of scss. Use `basis-1/4` and `basis-3/4` classes. I think you don't even need to set `flex-basis` of the right column if you specified the size of the left column. You can rely on flex-box and use `grow` class instead of specifying the exact column size using `basis-3/4` class.